.vscode/tasks.json

```diff
@@ -17,7 +17,38 @@
             "runOptions": {
                 "runOn": "folderOpen"
             },
-            "promptOnClose": true,
+            "promptOnClose": true
+        },
+        {
+            "label": "Build docker image",
+            "type": "docker-build",
+            "platform": "python",
+            "dockerBuild": {
+                "tag": "ctbbackend:latest",
+                "dockerfile": "${workspaceFolder}/Dockerfile",
+                "context": "${workspaceFolder}",
+                "pull": true
+            }
+        },
+        {
+            "label": "Run docker container (debug)",
```

Comment:
Maybe it's better to make these actions e.g. `.sh` scripts so that they're not dependent on IDE?

Reply:
I am afraid this configuration is largely dependent on VSCode and its docker extension (the debugging part). I guess people who don't use this OP editor must execute the command manually in a shell (and find a suitable debugger).

Also, as a VSCode task, it is platform independent unlike a bash script.